libs/core/src/lib/card/card.component.scss

```diff
@@ -17,6 +17,9 @@
     justify-content: flex-end;
     border-top: 0.0625rem solid #ccc;
```

Comment:
we can't have hard-coded colors like `#ccc`.

Reply:
at line 19, variable with value is used.
anyway, We have to move these styles to fundamental-styles.
will raise a different PR in fundamental-styles.

Reply:
@DeepakSap14 
There are 2 problems:
- using a hardcoded color value
- introducing css rules like this in ngx.

you can introduce this in styles(without hardcoded color value) and bring it from there. We are trying to reverse this practice.

Reply:
@DeepakSap14 I apologize for the hard-coded color, I am used to seeing 6 letter code for css variables like #cccccc and didn't pay attention to the next line of code.
@droshev it's actually correct, he is adding the fallback for IE11 support.